handbook/product/roles/index.md

```diff
@@ -1,20 +1,83 @@
-# Product team roles
-
-_Updated 2021-01-12_
-
-- [Product Manager](https://jobs.lever.co/sourcegraph/254299f5-f91b-43e2-aa1a-3732963dd296) - hiring +8 for the following teams grouped by priority of current business needs:
-  - Highest urgency
-    - [Core application](../../engineering/core-application/index.md)
-    - [Search](../../engineering/search/index.md)
-    - Spread code search/growth (new company effort)
-  - Medium urgency
-    - Backend platform (expansion from current Core application team growth)
-    - Frontend platform (expansion from current web team growth)
-    - [Distribution](../../engineering/distribution/index.md)
-  - Lowest urgency
-    - Integrations (expansion from current web team growth)
-    - [Security](https://jobs.lever.co/sourcegraph/c36db3e1-0ece-465d-ad7c-1eb6de9a4b22)
-- [Product Designer](https://jobs.lever.co/sourcegraph/fa7d3807-ae4c-4a35-9401-56dad0958227) - hiring +2
-- [Technical Writer](https://jobs.lever.co/sourcegraph/933f214e-338b-422b-b5d9-0b01057576d3) - hiring +2
-
-See our [careers page](../../../company/careers.md) for open positions and more info.
+# Roles on the Product team
+
+This page lists the roles on the product team.
+
+- Open product roles are listed on the [company careers page](../../../company/careers.md).
+- [Interview processes documentation](./interviews/index.md) has detailed steps for each role.
+- Because I am paranoid about losing data in Lever, I a Google Doc for each role with the full job description. These should be used for feedback from our team, and provide a backup data source. These docs are intentionally internal (please see the careers page linked above for open roles!). The docs are all located in this [Google Drive folder](https://drive.google.com/drive/folders/1a4yzjbqICYXMI7OLmDEDSK0BZD7EMydx?usp=sharing).
```

Comment:
```suggestion
- Because I am paranoid about losing data in Lever, I have a Google Doc for each role with the full job description. These should be used for feedback from our team, and provide a backup data source. These docs are intentionally internal (please see the careers page linked above for open roles!). The docs are all located in this [Google Drive folder](https://drive.google.com/drive/folders/1a4yzjbqICYXMI7OLmDEDSK0BZD7EMydx?usp=sharing).
```